io/disk/tiobench.py

```diff
@@ -64,10 +65,21 @@ def test(self):
                           each test should be averaged.
         """
         target = self.params.get('dir', default=self.workdir)
+        disk = self.params.get('disk', default=None)
+        fstype = self.params.get('fs', default='ext4')
         blocks = self.params.get('blocks', default=4096)
         threads = self.params.get('threads', default=10)
         size = self.params.get('size', default=1024)
         num_runs = self.params.get('numruns', default=2)
+
+        part_obj = Partition(disk, mountpoint=target)
+        self.log.info("Test will run on %s" % target)
+        self.log.info("Unmounting the disk/dir before creating file system")
+        part_obj.unmount()
+        self.log.info("creating file system")
+        part_obj.mkfs(fstype)
+        self.log.info("Mounting disk %s on directory %s" % (disk, target))
```

Comment:
Please notice that the recommended syntax for message string formatting in loggers is:
```python
        self.log.info("Mounting disk %s on directory %s", disk, target)
```
(https://docs.python.org/2/library/logging.html#logging.Logger.debug)